### PR TITLE
Installing composer in php 7.2

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/11020

image

### CODE REVIEW

**Comment by rageshkrishna** [comment]
The screenshot above shows composer working in php 7.1. What happened with 7.2?

**Comment by vijayreddy1991** [comment]
I will for php 7.2 also

image

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ available in the image:
       * redis
       * zmq
 
-  2. PHP 7.0.26 and PHP 7.1.12
+  2. PHP 7.0.30 and PHP 7.1.18
```
Comment (rageshkrishna) [inline README.md]: Are these extensions not installed for 7.2?

Comment (vijayreddy1991) [reply README.md]: updated

**Pull request merged**